docs/security.md

```diff
@@ -258,3 +258,31 @@ If you want to use the command `curl`, you can follow the commands:
+
+### Server configuration
+
+| Configuration item                                | Description                                                                                                                                                                                                                                   | Default value                                 | Required | Since version |
+|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|----------|---------------|
+| `gravitino.server.webserver.enableCorsFilter`     | Enable cross origin resource share filter.                                                                                                                                                                                                    | false                                         | No       | 0.4.0         |
+| `gravitino.server.webserver.allowedOrigins`       | A comma separated list of origins that are allowed to access the resources. Default value is *, meaning all origins.                                                                                                                          | `*`                                           | No       | 0.4.0         |
+| `gravitino.server.webserver.allowedTimingOrigins` | A comma separated list of origins that are allowed to time the resource. Default value is the empty string, meaning no origins.                                                                                                               | ``                                            | No       | 0.4.0         |
+| `gravitino.server.webserver.allowedMethods`       | a comma separated list of HTTP methods that are allowed to be used when accessing the resources. Default value is GET,POST,HEAD,DELETE.                                                                                                       | `GET,POST,HEAD,DELETE`                        | No       | 0.4.0         |
+| `gravitino.server.webserver.allowedHeaders`       | A comma separated list of HTTP headers that are allowed to be specified when accessing the resources. Default value is X-Requested-With,Content-Type,Accept,Origin. If the value is a single *, this means that any headers will be accepted. | `X-Requested-With,Content-Type,Accept,Origin` | No       | 0.4.0         |
+| `gravitino.server.webserver.preflightMaxAge`      | The number of seconds that preflight requests can be cached by the client. Default value is 1800 seconds, or 30 minutes.                                                                                                                      | `1800`                                        | No       | 0.4.0         |
```

Comment:
preflight is `pre` + `flight` or `perf` + `light`? The meaning of preflight is?

Reply:
pre + flight. This word is from the Jetty.

Reply:
It would be better to change to "gravitino.server.webserver.preflightMaxAgeInSecs"

Reply:
Ok.

Reply:
Fixed.